static/app/components/replays/videoReplayer.tsx

```diff
@@ -418,14 +428,19 @@ export class VideoReplayer {
   public pause(videoOffsetMs: number) {
     const index = this._currentIndex ?? 0;
 
+    // Hide first frame if we seek to a later part of the replay first
+    if (videoOffsetMs > 0) {
+      this.hideVideo(0);
+    }
```

Comment:
Will this hide (or flash) the video if we seek to somewhere else in the first segment?

Reply:
it looks all right to me:
https://github.com/getsentry/sentry/assets/56095982/5c5b80d4-b3e9-483f-afcf-aa113268348a